OCR/frontend/e2e/App.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { test, expect } from '@playwright/test';
 import {Simulate} from "react-dom/test-utils";
 import click = Simulate.click;
```

Comment:
this is not used

Reply:
I hate it when my editor tries to be helpful and adds in new imports and it seems like it can detect they are unused but not remove them. Removed.